cmds/clients/contestcli-http/main.go

```diff
@@ -77,26 +81,56 @@ func run(verb string) error {
 			return fmt.Errorf("failed to parse job descriptor: %v", err)
 		}
 		params.Add("jobDesc", string(jobDesc))
+		resp, err := request(verb, params)
+		if err != nil {
+			return err
+		}
+		fmt.Println(resp)
+		// if wait flag set, keep polling for status till job is completed
+		if *flagWait {
+			fmt.Fprintf(os.Stderr, "\nWaiting for job to complete...")
```

Comment:
this breaks the assumption that this program only prints JSON output. I understand that `-wait` implies that we cannot print a single JSON document, but maybe this could print one-line JSON documents instead?

Reply:
I think printing one line JSON is still not going to be consistent with the rest of the operations. Also changing the output of all other operations will be disruptive and beyond the purpose of a confined change to add a flag. Maybe for -wait we can skip printing the start response? Or we can just highlight this difference in the flag help.

Reply:
highlighting the difference in the help sounds good